### PR TITLE
Update general documentation to comply with style guides

### PR DESCRIPTION
**Pull request by ppbaena**
### Description of the change

Update files in the root folder to comply with [documentation style guides](https://github.com/vmware-tanzu/kubeapps/blob/main/site/content/docs/latest/reference/developer/style-guide.md#documentation-style-guide)

- ADOPTERS.md
- CODE_OF_CONDUCT.md
- CONTRIBUTING.md
- GOVERNANCE.md
- MAINTAINERS.md
- README.md
- ROADMAP.md
- SECURITY.md

### Benefits

Documentation updated and easy to follow and maintain by following the style guides.

### Possible drawbacks

None.

### Applicable issues

- related to #5024 

### Additional information

This is the first PR to update all documentation according to style guides (tutorials, how-to guides, background and reference will follow).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
 
 Please familiarize yourself with the [Code of Conduct](https://github.com/vmware/.github/blob/main/CODE_OF_CONDUCT.md) before contributing.
 
-* _CLA: Before you start working with Kubeapps, please read and sign our Contributor License Agreement [CLA](https://cla.vmware.com/cla/1/preview). If you wish to contribute code and you have not signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq)._
+- _CLA: Before you start working with Kubeapps, please read and sign our Contributor License Agreement [CLA](https://cla.vmware.com/cla/1/preview). If you wish to contribute code and you have not signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq)._
 
 ## Ways to contribute
 
 We welcome many different types of contributions and not all of them need a Pull request. Contributions may include:
```
Comment (castelblanque) [inline CONTRIBUTING.md:11]:
```suggestion
Kubeapps project welcomes many different types of contributions and not all of them need a Pull request. Contributions may include:
```
?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -50,43 +50,43 @@ For specifics on what to include in your report, please follow the guidelines in
...
+To suggest a feature, please first open an [issue](https://github.com/vmware-tanzu/kubeapps/issues/new?assignees=&labels=kind%2Ffeature&template=feature_request.md&title=) that will be tagged with [`kind/proposal`](https://github.com/vmware-tanzu/kubeapps/labels/kind%2Fproposal), or create a new [Discussion](https://github.com/vmware-tanzu/kubeapps/discussions/new). The project maintainers will work with you on your feature request.
 
 Once the feature request has been validated, a pull request can be opened to implement the feature.
 
 For specifics on what to include in your feature request, please follow the guidelines in the issue and pull request templates.
 
 ## Ask for Help
 
-The best way to reach us with a question when contributing is to ask on:
+The best way to reach kubeapps maintainers team with a question when contributing is to ask on:
```
Comment (castelblanque) [inline CONTRIBUTING.md:77]:
```suggestion
The best way to reach Kubeapps maintainers team with a question when contributing is to ask on:
```

File: ROADMAP.md [hunk 3]
```diff
@@ -16,23 +16,21 @@ Discussion on the roadmap can take place in threads under [Issues](https://githu
...
 
-If you don't know where to start, we are always looking for contributors that will help us reduce technical, automation, and documentation debt. Please take the timelines & dates as proposals and goals. Priorities and requirements change based on community feedback, roadblocks encountered, community contributions, etc. If you depend on a specific item, we encourage you to contact us by Slack, or help us deliver that feature by contributing to Kubeapps.
+If you don't know where to start, kubeapps maintainers are always looking for contributors that will help to reduce technical, automation, and documentation debt. Please take the timelines & dates as proposals and goals. Priorities and requirements change based on community feedback, roadblocks encountered, community contributions, etc. If you depend on a specific item, kubeapps maintainers team encourage you to contact them by Slack, or help them deliver that feature by contributing to Kubeapps.
```
Comment (castelblanque) [inline ROADMAP.md:27]:
```suggestion
If you don't know where to start, Kubeapps maintainers are always looking for contributors that will help to reduce technical, automation, and documentation debt. Please take the timelines & dates as proposals and goals. Priorities and requirements change based on community feedback, roadblocks encountered, community contributions, etc. If you depend on a specific item, kubeapps maintainers team encourage you to contact them by Slack, or help them deliver that feature by contributing to Kubeapps.
```
Kubeapps starting with capital letter?

**Review by castelblanque — APPROVED** [review]
Awesome, many thanks for the update!

**Comment by ppbaena** [comment]
Thanks for double checking @castelblanque !

**Pull request merged**